hq/app/schedule/IamJob.scala

```diff
@@ -11,11 +12,11 @@ import utils.attempt.FailedAttempt
 
 import scala.concurrent.ExecutionContext
 
-class IamJob(enabled: Boolean, cacheService: CacheService, snsClients: AwsClients[AmazonSNSAsync])(executionContext: ExecutionContext) extends JobRunner with Logging {
+class IamJob(enabled: Boolean, cacheService: CacheService, snsClients: AwsClients[AmazonSNSAsync], config: Configuration)(executionContext: ExecutionContext) extends JobRunner with Logging {
   override val id = "credentials report job"
   override val description = "Automated emails for old permanent credentials"
-  override val cronSchedule: CronSchedule = CronSchedules.onceADayAt1am
-  val topicArn: String = ??? //TODO retrieve from config
+  override val cronSchedule: CronSchedule = CronSchedules.firstMondayOfEveryMonth
+  val topicArn: String = getAnghammaradSNSTopicArn(config).getOrElse("") //TODO in the event that it is a None, what should we do?
```

Comment:
How would it be best to handle this?

Reply:
generally with features like notifications we would not send notifications locally by default, not having the config there is one way of making this happen

Reply:
Great, I've made this change thanks 👍🏽